### PR TITLE
Add dashboards to be managed by Terraform.

### PR DESCRIPTION
**Pull request by yegle**
NOTE: the chart on metric
custom.googleapis.com/opencensus/en-verification-server/ratelimit/limitware/rate_limited_count
was removed as it's no longer exported by the code.

The API requires JSON config but writing multi-line strings in JSON is PITA. I've decided to write it in YAML which is more compact and easier to write multi-line queries.

Related issue: #460 

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by sethvargo** [comment]
This looks like it includes some changes from your other PR - is that intended?

File: terraform/alerting/dashboards/e2e.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "displayName": "e2e",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "OpenCensus/en-verification-server/api/issue/codes_issued_count (filtered) (gr...",
```
Comment (sethvargo) [inline terraform/alerting/dashboards/e2e.json:7]: ...?

Comment (yegle) [reply terraform/alerting/dashboards/e2e.json]: This JSON was exported from existing dashboards. Updated to have a better name.

**Comment by yegle** [comment]
> This looks like it includes some changes from your other PR - is that intended?

That was intended. I was trying to chain the changes and base this change on my other unmerged changes.

It's ready to review now.

**Comment by yegle** [comment]
Gentle ping?

File: terraform/main.tf [hunk 2]
```diff
@@ -53,6 +53,8 @@ resource "google_project_service" "services" {
     "sqladmin.googleapis.com",
     "storage.googleapis.com",
     "vpcaccess.googleapis.com",
+    "monitoring.googleapis.com",
```
Comment (sethvargo) [inline terraform/main.tf:56]: Nit, can you sort this list?

**Review by sethvargo — COMMENTED** [review]
One nit

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**